python_modules/dagster/dagster/_core/definitions/asset_spec.py

```diff
@@ -204,6 +211,7 @@ def dagster_internal_init(
             automation_condition=automation_condition,
             owners=owners,
             tags=tags,
+            kinds=kinds,
```

Comment:
I would limit this to 2 with a hard error until we support multiple kinds in the UI